docs/index.md

```diff
@@ -26,5 +27,6 @@ Documentation for releases before joining Apache have been [archived](archive).
 [fluo-1.2]: /docs/fluo/1.2/
 [fluo-1.1]: /docs/fluo/1.1.0-incubating/
 [fluo-1.0]: /docs/fluo/1.0.0-incubating/
+[recipes-1.2]: /docs/fluo-recipes/1.2/
```

Comment:
This page doesn't exist yet. I'm not sure if anything has changed since 1.1, but we probably don't want to get into the habit of copy/pasting unchanged documentation to get the URLs to work out. I'm not sure what the best thing to do here is. IMO, ideally, we'd have a single set of 1.x documentation, and we'd just annotate a "since 1.1", "since 1.2", etc., as appropriate. I think that's the direction Accumulo's website is going for Accumulo 2.x.

Reply:
Next week I am going to look into the docs for version 1.2

Reply:
To be honest I'm not as familiar with the website code as I'd like to be. The reason for this pull request was that I was on this page: [https://fluo.apache.org//docs/](https://fluo.apache.org//docs/). I clicked the most recent version of the fluo-recipe docs listed which is 1.1.0-incubating. That brought me here: [https://fluo.apache.org/docs/fluo-recipes/1.1.0-incubating/](https://fluo.apache.org/docs/fluo-recipes/1.1.0-incubating/). At the top of that page there is a banner that says "These docs are for Fluo Recipes 1.1.0-incubating which is an old version! Check out the latest docs!" The banner provides a link here: [https://fluo.apache.org//docs/fluo-recipes/1.2/getting-started/overview](https://fluo.apache.org//docs/fluo-recipes/1.2/getting-started/overview), which seems to be the 1.2 fluo-recipe docs.

It seemed like a link to [https://fluo.apache.org//docs/fluo-recipes/1.2/getting-started/overview](https://fluo.apache.org//docs/fluo-recipes/1.2/getting-started/overview) should be added to [https://fluo.apache.org//docs/](https://fluo.apache.org//docs/), but again I'm not as familiar with the website code as I'd like to be so I'm not sure if that's accurate or what the best way of accomplishing that is.

Reply:
The page https://fluo.apache.org/docs/fluo-recipes/1.2 definitely exists.  Its not at that location in source though, in source its in the `_recipes-1-2` dir and I think something in `_config.yml` maps `/docs/fluo-recipes/1.2` to `_recipes-1-2`.  I think this change is appropriate given the docs already exists.

Reply:
Oh, I see. In that case, then :+1: for this change.